src/test/container-features/helpers.test.ts

```diff
@@ -1,274 +0,0 @@
-import { assert } from 'chai';
```

Comment:
This was originally the contents of `featureHelpers.test.ts` that I renamed and refactored